### ISSUE TITLE
\c invalid_database 

### ISSUE THREAD

**Issue by fzerorubigd** [amjith/pgcli#76]
Using \c with an invalid database name result in crash. 
```
postgres> \c aaa
Traceback (most recent call last):
  File "/usr/local/bin/pgcli", line 9, in <module>
    load_entry_point('pgcli==0.10.2', 'console_scripts', 'pgcli')()
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 610, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 590, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 782, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 416, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/pgcli/main.py", line 119, in cli
    click.secho(e.pgerror, err=True, fg='red')
  File "/usr/local/lib/python2.7/dist-packages/click/termui.py", line 381, in secho
    text = style(text, **styles)
  File "/usr/local/lib/python2.7/dist-packages/click/termui.py", line 354, in style
    return ''.join(bits)
TypeError: sequence item 1: expected string, NoneType found
```

**Comment by amjith** [comment]
Yup! I found out about it earlier today. :(

This will be fixed in the next release.

**Comment by amjith** [comment]
This now fixed by 803e460.

**Comment by amjith** [comment]
Fixed in 0.10.3. 

https://pypi.python.org/pypi/pgcli/0.10.3
